src/containers/Network/UpgradeStatus.tsx

```diff
@@ -22,63 +22,115 @@ import {
 } from '../shared/vhsTypes'
 import NetworkContext from '../shared/NetworkContext'
 import { ledgerCompare } from './Nodes'
+import { Loader } from '../shared/components/Loader'
 
-interface DataAggregation {
-  label: string
-  validatorsPercent: number
-  validatorsCount: number
+interface NodesStats {
```

Comment:
These types are a bit harder to read with the first word pluralized, and I don't think the pluralization adds anything - can you remove it? i.e. `NodeStats` instead of `NodesStats`.